pelican/plugins/image_process/image_process.py

```diff
@@ -342,7 +341,7 @@ def harvest_images_in_fragment(fragment, settings):
         elif not isinstance(d, dict):
             raise RuntimeError(
                 "Derivative %s definition not handled"
-                "(must be list or dict)" % (derivative)
```

Comment:
Can you do this? I've run into cases where a tuple (or list) needs to be passed and can never remember exactly when it's required...

Reply:
Yes this works. A tuple is required if there is multiple `%` parameters in the string.